pkg/virtctl/console/console.go

```diff
@@ -0,0 +1,138 @@
+package console
+
+import (
+	"fmt"
+	"github.com/gorilla/websocket"
+	flag "github.com/spf13/pflag"
+	"golang.org/x/crypto/ssh/terminal"
+	"io"
+	"k8s.io/client-go/pkg/api/v1"
+	"k8s.io/client-go/tools/clientcmd"
+	"log"
+	"net/url"
+	"os"
+	"os/signal"
+	"time"
+)
+
+type Console struct {
+}
+
+func (c *Console) FlagSet() *flag.FlagSet {
+	cf := flag.NewFlagSet("console", flag.ExitOnError)
+	cf.StringP("device", "d", "serial0", "Console to connect to")
+
+	return cf
+}
+
+func (c *Console) Usage() string {
+	usage := "Connect to a serial console on a VM:\n\n"
+	usage += "Examples:\n"
+	usage += "#Connect to the console 'serial0' on the VM 'myvm':\n"
```

Comment:
Yes, but a whitespace was missing. Have for instance a look at `kubectl get --help` to see how they are formatting the help pages.